modules/nested-tabs/assets/js/frontend/handlers/nested-tabs.js

```diff
@@ -325,4 +332,117 @@ export default class NestedTabs extends Base {
+			keyEscape = 27,
+			isTabPressed = keyTab === event?.which,
+			isShiftPressed = event?.shiftKey,
+			isShiftAndTabPressed = !! isTabPressed && isShiftPressed,
+			isOnlyTabPressed = !! isTabPressed && ! isShiftPressed,
+			isEscapePressed = keyEscape === event?.which;
+
+		if ( isShiftAndTabPressed ) {
+			return 'ShiftTab';
+		} else if ( isOnlyTabPressed ) {
+			return 'Tab';
+		} else if ( isEscapePressed ) {
+			return 'Escape';
+		}
+	}
+
+	changeFocusFromContentContainerItemBackToTabTitle( event ) {
+		if ( this.hasDropdownLayout() ) {
+			return;
+		}
+
+		const isShiftAndTabPressed = 'ShiftTab' === this.getKeyPressed( event ),
+			isOnlyTabPressed = 'Tab' === this.getKeyPressed( event ),
+			isEscapePressed = 'Escape' === this.getKeyPressed( event ),
+			firstItemIsInFocus = this.itemInsideContentContainerHasFocus( 0 ),
+			lastItemIsInFocus = this.itemInsideContentContainerHasFocus( 'last' ),
+			activeTabTitleFilter = `.${ this.getActiveClass() }`,
+			$activeTabTitleVisible = this.getVisibleTabTitle( activeTabTitleFilter ),
+			activeTabTitleIndex = parseInt( $activeTabTitleVisible?.getAttribute( 'data-tab' ) ),
+			nextTabTitleFilter = this.getTabTitleFilterSelector( activeTabTitleIndex + 1 ),
+			$nextTabTitleVisible = this.getVisibleTabTitle( nextTabTitleFilter ),
+			pressShiftTabOnFirstFocusableItem = isShiftAndTabPressed && firstItemIsInFocus && !! $activeTabTitleVisible,
+			pressTabOnLastFocusableItem = isOnlyTabPressed && lastItemIsInFocus && !! $nextTabTitleVisible;
+
+		if ( ( pressShiftTabOnFirstFocusableItem ) || isEscapePressed ) {
+			event.preventDefault();
+
+			$activeTabTitleVisible?.focus();
+		} else if ( pressTabOnLastFocusableItem ) {
+			event.preventDefault();
+
+			this.setTabindexOfActiveContainerItems( '-1' );
+
+			$nextTabTitleVisible?.focus();
+		}
+	}
+
+	changeFocusFromActiveTabTitleToContentContainer( event ) {
+		const isOnlyTabPressed = 'Tab' === this.getKeyPressed( event ),
+			$focusableItems = this.getFocusableItemsInsideActiveContentContainer(),
+			$firstFocusableItem = $focusableItems[ 0 ],
+			$currentTabTitle = elementorFrontend.elements.window.document.activeElement,
+			currentTabTitleIndex = parseInt( $currentTabTitle?.getAttribute( 'data-tab' ) );
+
+		if ( isOnlyTabPressed && this.tabTitleHasActiveContentContainer( currentTabTitleIndex ) && !! $firstFocusableItem ) {
+			event.preventDefault();
+			$firstFocusableItem.focus();
+		}
+	}
+
+	itemInsideContentContainerHasFocus( position ) {
+		const $currentItem = elementorFrontend.elements.window.document.activeElement,
+			$focusableItems = this.getFocusableItemsInsideActiveContentContainer(),
+			itemIndex = 'last' === position ? $focusableItems.length - 1 : position;
+
+		return $focusableItems[ itemIndex ] === $currentItem;
+	}
+
+	getFocusableItemsInsideActiveContentContainer() {
+		const settings = this.getSettings();
+
+		return this.$element.find( settings.selectors.activeTabContentContainers ).find( ':focusable' );
```

Comment:
If `getFocusableItemsInsideActiveContentContainer` is set in the mega menu only. what is the reason that this function and all the other functions that are dependent on this one are here and not in the mega menu.

Reply:
I have moved all functions which will be necessary for the nested tabs accessibility to the nested tabs handler.
We will need this in the next task.